include/ppx/grfx/grfx_config.h

```diff
@@ -57,6 +57,7 @@ class Swapchain;
 class TextDraw;
 class Texture;
 class TextureFont;
+class YcbcrConversion;
```

Comment:
So originally, I had it named that, but because I'm using it (in grfx) with Texture, and it maps to Sampler + SampledImage, and maybe other constructs in other languages, I shortened the name. If preferred, I can add this back, but let me know what your thoughts are based on that context!

Reply:
I could also potentially name it 'YcbcrColorConversion'?

Reply:
Re-read the code. What do you think about just keeping SamplerYcbcrConversion since that maps closely to the underlying API construct and that way it is easier to figure out what it corresponds? I don't think we need to overthink as to what it might map to in the future. Otherwise I am fine with the existing name as well.

Reply:
Renamed to SamplerYcbcrConversion